src/checksum_verify.ts

```diff
@@ -0,0 +1,20 @@
+import * as core from "@actions/core";
+import * as crypto from "crypto"
+import * as fs from "fs"
+
+export function checksumVerify(downloadPath: string){
```

Comment:
@h0x0er name should be in form of verbNoun, so in this case it would be verifyChecksum. Observe the other method names, like createHash, readFile etc.  

Also, the file name can just be checksum.ts

Reply:
@varunsh-coder fixed the naming problem.